Sources/Drill/FileHandle+TextOutputStream.swift

```diff
@@ -7,11 +7,5 @@
 
 import Foundation
 
-extension FileHandle: TextOutputStream {
```

Comment:
We really oughta figure out what owns this.

Reply:
I kinda want to put `Writer<T>` in Lithosphere.